packages/utils/src/peripherals/chainData.ts

```diff
@@ -63,6 +63,7 @@ export type ChainData = {
     standard: string;
   }[];
   gasEstimates: {
+    execute: string;
     prepare: string;
     fulfill: string;
     cancel: string;
```

Comment:
we can remove prepare/fulfill/cancel because we don't use them anymore in amarok